themes/bootstrap3/templates/RecordTab/holdingsils.phtml

```diff
@@ -123,12 +123,12 @@
         try {
             echo $this->context($this)->renderInContext(
                 'RecordTab/holdingsils/' . $this->tab->getTemplate() . '.phtml',
-                ['holding' => $row]
+                ['holding' => $row, 'callNos' => $callNos]
```

Comment:
There has to be a better way to do this, but two levels deep in `boundwith.phtml` I want to format this item differently based on whether there are multiple call numbers being displayed.  In `holdingsils.phtml` the `$holding` variable can provide this info, but that name is being overridden on this line by the specific row, so it's no longer accessible.

Reply:
This seems like a case of bad variable naming. It would make more sense if 'holding' were called 'item' in this context, and then you could also pass in the true holdings record as 'holding'. It's unfortunate that this wasn't noticed much earlier! That might be a knot worth untangling eventually, but I think under the circumstances, just passing down the call numbers isn't a terrible solution. Since work was already done to process and deduplicate them, it seems better to provide them to the template instead of recalculating them, even if the subtemplate did have access to full holdings.